mentoring/programs/contributor-workshop/resources/guides/videoguide.md

```diff
@@ -0,0 +1,397 @@
+    - [Zoom Recording](#zoom-recording)
+    - [OBS](#obs)
+    - [Streamyard](#streamyard)
+    - [Adobe Media Encoder](#adobe-media-encoder)
+  - [Recording Guides](#recording-guides)
+    - [Recording in Zoom](#recording-in-zoom)
+      - [Install Windowed extension (optional)](#install-windowed-extension-optional)
+      - [Start the meeting](#start-the-meeting)
+      - [Setting up slides for recording](#setting-up-slides-for-recording)
+        - [Without browser extension](#without-browser-extension)
+        - [With browser extension](#with-browser-extension)
+          - [Note : Exiting the windowed mode](#note--exiting-the-windowed-mode)
+      - [Adding your facecam](#adding-your-facecam)
+      - [Unmute your audio](#unmute-your-audio)
+      - [Recording in Zoom](#recording-in-zoom-1)
+        - [Note : Do a test recording first](#note--do-a-test-recording-first)
+        - [How to record](#how-to-record)
+    - [Recording in OBS](#recording-in-obs)
+    - [Recording in Streamyard](#recording-in-streamyard)
+  - [Updating / Adding video to a segment](#updating--adding-video-to-a-segment)
+    - [Rename your recording](#rename-your-recording)
+    - [Update Google Drive](#update-google-drive)
+      - [Uploading your video](#uploading-your-video)
+        - [What folder to upload to](#what-folder-to-upload-to)
+    - [Updating Youtube](#updating-youtube)
+      - [Upload the new video](#upload-the-new-video)
+        - [Adding the thumbnail image](#adding-the-thumbnail-image)
+        - [Setting the description](#setting-the-description)
+      - [Update the previous video for the segment if it exists](#update-the-previous-video-for-the-segment-if-it-exists)
+        - [Update the title](#update-the-title)
+        - [Update the thumbnail image](#update-the-thumbnail-image)
+        - [Update the Description](#update-the-description)
+        - [Update the Comments](#update-the-comments)
+      - [Updating the Segment page with video](#updating-the-segment-page-with-video)
+  - [Best Practices & Recommendations](#best-practices--recommendations)
+    - [Video](#video)
+      - [Camera Type](#camera-type)
+    - [Audio](#audio)
+      - [Microphone Types](#microphone-types)
+      - [Sound Environment](#sound-environment)
+        - [Background Noise](#background-noise)
+        - [Other Tips](#other-tips)
+    - [Screensharing](#screensharing)
+    - [Hardware](#hardware)
+      - [Headset Microphones](#headset-microphones)
+      - [External Microphones](#external-microphones)
+      - [Webcams](#webcams)
+      - [Presentation remotes](#presentation-remotes)
+
+## Format / Video Style
+
+You can use any format/style as long as the same information in the guide is captured in the video. Slides, A screenshare, greenscreen yourself into the repo even! Be creative.
+
+## Encoding Guidelines
+
+### Zoom Recording
+Handled by zoom, Can't change these settings.
+
+### OBS
+(todo : match with media encoder)
```

Comment:
make this a bulletted list

Reply:
![image](https://user-images.githubusercontent.com/19580905/117864830-240e9380-b28d-11eb-88e6-ce609f56909a.png)